app/code/Magento/Catalog/Test/Mftf/Test/AdminMassProductPriceUpdateTest.xml

```diff
@@ -25,56 +25,64 @@
         </before>
         <after>
             <deleteData createDataKey="simpleProduct1" stepKey="deleteSimpleProduct1"/>
-            <deleteData createDataKey="simpleProduct2" stepKey="deleteSimpleProduct2"/>
+            <deleteData createDataKey="simpleProduct2" stepKey="deleteSimpleProduct2"/> 
```

Comment:
Please, remove the trailing space at the end.

Reply:
Removed, thank you.